### DISCUSSION TITLE
Should I go ahead and try to export the project again?

### DISCUSSION THREAD

**Discussion by 1natalie** [leightonavantgardehaus/.github#7] [Q&A] [answered]
I would assume I need to add it to all the other projects we have in have in the solution file ? 

 restore the package
build the solution then 
/microsoft/vstest-action with the specific assembly for the corresponding folder

**Comment by TL8N** [comment, marked as answer]
Try to use vstest.console.exe command line tool to run the test by specifying the assembly path

vstest.console.exe

You can download the vstest.console.exe binary and include it in your repository. Then you can reference it in your GitHub Action workflow and run the tests using vstest.console.exe.

Ensure that the environment variable TEST_BUILD is set correctly and that it contains the path to the test build you want to run.

Verify that the necessary dependencies, such as the .NET Framework, are installed on the machine where the command is being executed.

Try running the command with the /diag option to generate more detailed diagnostic output that may give more information about what's causing the error ! 

Hope it helps
